src/set_theory/ordinal_arithmetic.lean

```diff
@@ -1428,23 +1428,29 @@ lemma enum_ord_def_nonempty (hS : unbounded (<) S) {o} :
 @[simp] theorem enum_ord_zero : enum_ord S 0 = Inf S :=
 by { rw enum_ord_def, simp [ordinal.not_lt_zero] }
 
-theorem enum_ord_zero_le {a} (ha : a ∈ S) : enum_ord S 0 ≤ a :=
```

Comment:
Are you intentionally dropping this?

Reply:
I reasoned that it didn't make much sense to have lemmas about `enum_ord S 0` since that would get simped out into the much simpler `Inf S` anyways.